features/proposed/add-location-services/README.md

```diff
@@ -0,0 +1,71 @@
+# Integrate Location Services to FxA
+
+## Problem Summary
+
+Currently, FxA does not utilize any location data for a user. When a user logins into the system, location data is not used to determine whether that login is fraudulent or not. Furthermore, once a user has logged in, the notification email does not give an approximate location where it occurred. By utilizing location data, users can take the appropriate measure (reset password, remove device, etc) to secure their account.
+
+Although location data can be derived after the fact, it would take longer to surface this information to a user. In the waiting process, the user’s account could have already been compromised.
+
+### Assumptions
+
+According to [Maxmind](https://www.maxmind.com/en/geoip2-city), it claims 99.8% accurate on a country level, 90% accurate on a state level, 81% accurate on a city level for the US within a 50 kilometer radius. Outside of the US it can be considerably lower.
+
+## Outcomes
+
+The primary outcome for the feature would be the ability to add user location data to FxA. 
+This in turn will be used to add that information on emails and help detect fraudulent account access.
+
+## Hypothesis
+
+We believe that adding location services will provide more security and a better user experience. 
+We will know this to be true by monitoring feedback of user’s.
+
+## Metrics
+
+Summary of metrics that will be monitored
```

Comment:
@TDA @rfk I thought it would be useful for us to monitor the accuracy of the location data (as reported by maxmind). However, I am open to what we define as accuracy/inaccurate. Took a first stab at these definitions.